queries/python/highlights.scm

```diff
@@ -9,7 +9,7 @@
 
 ;; Identifier naming conventions
 ((identifier) @type
- (#match? @type "^[A-Z]"))
+ (#match? @type "^[A-Z].*[a-z]"))
 ((identifier) @constant
  (#match? @constant "^[A-Z][A-Z_0-9]*$"))
```

Comment:
isn't this overridden by this query already?

![Screenshot from 2021-06-29 14-36-21](https://user-images.githubusercontent.com/4975310/123857024-68123200-d8e7-11eb-8e2f-1e6000004b7b.png)

Reply:
It's overwritten if the highlight of constant is strong enough to overwrite type. Actually both of them are applied at the same time only Constant over type. So when Type is bold and Constant not bold Constant can not overwrite the boldness of Type just the color. I guessed that this was the issue of the use.

Reply:
@gjeusel is there a reason why constant couldn't visually overwrite `@type` for you. Italics? Something fancy or just because it's displayed in `:TSHighlightsUnderCursors`

Reply:
Something fancy, setting `hi! link pythonTSConstant white`, hence defaulting the constant highlight to the type one.